news/11740.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+Reconcile computation of isolated build environment paths. Fix all
```

Comment:
I'm wary of saying "all" here.

Id change this to something like:

> Improve handling of isolated build environments, on interpreters with significantly modified interpreter paths.

Reply:
> this is more detail than I'd like for end-user communication.

This is still the case.

Reply:
Reworded again.